### DISCUSSION TITLE
Expand the data export fields

### DISCUSSION THREAD

**Discussion by jumson** [simple-login/app#439] [Feature Requests]
Would like to see historical data, including the record of all Forwards, Blocks, and Replies, and other similar information that is accessed already in the UI. (The logs, I guess)
Current data export feature exports current aliases, mailboxes, domains.

Google, Twitter, Facebook, etc. all have similar features done by request and usually takes a day.

**Comment by EthraZa** [comment]
I was just looking for that. 

The UI already shows in More:
```php
0 forwards, 0 blocks, 0 sents
```
Also in See All in More:
```php
0 Email Handled, 0 Email Forwarded, 0 Email Replied, 0 Email Blocked
```

Why is this data not being exported in the CSV data at **Export Aliases** function?

**Reply by nguyenkims** [reply]
The data exported can be imported later and that's why temporary data like stats aren't exported.

**Reply by EthraZa** [reply]
Can this data be retrieved over some API endpoint?

**Reply by nguyenkims** [reply]
Yeah, you can use this endpoint https://github.com/simple-login/app/blob/master/docs/api.md#get-apiv2aliases to get these info.

**Comment by EthraZa** [comment]
I don 't know why I was unable to find the [API description](https://github.com/simple-login/app/blob/master/docs/api.md#alias-endpoints) at the DOCs or even Google, I found this only last night before to sleep, very deep in a subreddit.
 IMHO it would deserve a menu entry of its own in the DOCs. 
But now it's found, let's explore it. 
Tks

**Comment by EthraZa** [comment]
Hi @nguyenkims ,
I'm already accessing the [API ](https://github.com/simple-login/app/blob/master/docs/api.md#get-apiv2aliases) to get **nb_forward** that is what I need to know for each alias - But, for what period nb_ data refers to? Is it just the sum of the last 14 days?

**Reply by nguyenkims** [reply]
Yes the `nb_` data refers to the last 14 days stats.
